### PR TITLE
Task #111639 fix: Update paypal IPN verification URL

### PR DESCRIPTION
**Pull request by ankush-maherwal**

### CODE REVIEW

File: paypal/paypal/helper.php [hunk 1]
```diff
@@ -88,7 +88,9 @@ public function Storelog($name, $logdata)
 	 */
 	public function validateIPN($data, $componentName)
 	{
-		$url              = self::buildPaypalUrl();
+		$sandBoxMode = $this->params->get('sandbox', '0', 'INT');
+		$url = ($sandBoxMode)?'https://ipnpb.sandbox.paypal.com/cgi-bin/webscr':'https://ipnpb.paypal.com/cgi-bin/webscr';
```
Comment (manojLondhe) [inline paypal/paypal/helper.php]: Can you make it more readable like
`$url = ($sandBoxMode) ? 'https://ipnpb.sandbox.paypal.com/cgi-bin/webscr' : 'https://ipnpb.paypal.com/cgi-bin/webscr';`

@ankush-maherwal

**Comment by manojLondhe** [comment]
@ankush-maherwal put summary of chages

File: paypal/paypal/helper.php (hunk 1 above)
Comment (ankush-maherwal) [reply paypal/paypal/helper.php]: done

File: paypal/paypal/helper.php [hunk 2]
```diff
@@ -86,7 +86,9 @@ public function Storelog($name, $logdata)
 	 */
 	public function validateIPN($data, $componentName)
 	{
-		$url              = self::buildPaypalUrl();
```
Comment (manojLondhe) [inline paypal/paypal/helper.php]: so we not using `buildPaypalUrl` anymore?
can we use it `buildPaypalUrl` and modify it as needed? @ankush-maherwal

Comment (ankush-maherwal) [reply paypal/paypal/helper.php]: We are using function 'buildPaypalUrl', this function creates URL for posting data to PayPal for payment.

As PayPal has provided a separate server for IPNs we are not using function 'buildPaypalUrl' to create IPN URL.

Also, we are using IPN URL only in one place hence not created a separate function for this.

**Pull request merged**